src/dir/glob.cr

```diff
@@ -166,6 +166,7 @@ class Dir
         when EntryMatch
           return if sequence[pos + 1]?.is_a?(RecursiveDirectories)
           each_child(path) do |entry|
+            next if entry[0] == '.' && !options[:match_hidden]
```

Comment:
```suggestion
            next if !options[:match_hidden] && entry.starts_with?('.')
```

Reply:
Updated both cases this code was present.